### ISSUE TITLE
Error running command sfdx plugins:install @salesforce/lwc-dev-server

### ISSUE THREAD

**Issue by ganieany52** [forcedotcom/lwc-dev-server-feedback#112]
**Describe the bug**
Receiving the below error when running the command `sfdx plugins:install @salesforce/lwc-dev-server` in the terminal

```
Output:
node-gyp not found! Please ensure node-gyp is in your PATH--
Try running: `sudo npm install -g node-gyp`
spawn node-gyp ENOENT
/bin/sh: nodejs: command not found
Installing plugin @salesforce/lwc-dev-server... failed

```
When I run the suggested command, it returns `sudo: npm: command not found`

**Expected behavior**
Successful install 

**Desktop (please complete the following information):**
 - MacOS Catalina & Google Chrome

**Additional context**
Have tried on two separate machines with the same error

**Comment by ganieany52** [comment]
Got it working by installing node.js https://nodejs.org/en/ and then running the command recommended in the error message `sudo npm install -g node-gyp`. Then when I ran `sfdx plugins:install @salesforce/lwc-dev-server` again my machine actually prompted me to install Xcode Command Line Tools. After installing that, I ran `sfdx plugins:install @salesforce/lwc-dev-server` one last time and it worked!
